src/python/pants/goal/run_tracker.py

```diff
@@ -134,7 +134,7 @@ def register_options(cls, register):
             help="Option scopes to record in stats on run completion. "
             "Options may be selected by joining the scope and the option with a ^ character, "
             "i.e. to get option `pantsd` in the GLOBAL scope, you'd pass "
-            "`GLOBAL^pantsd`",
+            "`GLOBAL^pantsd`, add a '*' to the list to capture all known scopes",
```

Comment:
Nit: I'd make this a separate sentence. "`GLOBAL^pantsd`. Add a '*' to the list to capture all known scopes",